oxide.json

```diff
@@ -9812,6 +9812,60 @@
           "size"
         ]
       },
+      "DiskImport": {
```

Comment:
so: this file comes untouched from omicron/openapi/nexus.json -- we can't modify it locally. It gets automatically updated.

Reply:
I really like the idea of generating the builder for this type. Does it make sense to either 1) Make the same change in nexus.json or 2) add another openAPI specification or generating internal types?